packages/common/src/hardhat/tasks/identifiers.js

```diff
@@ -28,13 +31,32 @@ task("whitelist-identifiers", "Whitelist identifiers from JSON file")
 task("migrate-identifiers", "Adds all whitelisted identifiers on one IdentifierWhitelist to another")
   .addParam("from", "The contract from which to query a whitelist of identifiers.", "", types.string)
   .addOptionalParam("to", "The contract on which to whitelist new identifiers.", "", types.string)
+  .addOptionalParam(
```

Comment:
Unrelated to core PR but this adds feature to migrate identifiers cross-chain. Useful for setting up polygon identifier whitelists